hotspot/test/runtime/NMT/HugeArenaTracking.java

```diff
@@ -25,6 +25,7 @@
  * @test
  * @key nmt jcmd
  * @library /testlibrary /testlibrary/whitebox
+ * @requires vm.bits == 64
```

Comment:
I don't think this has any effect on JDK 8. AFAIK, `vm.bits` is being set in JDK 11+ via `test/jtreg-ext/requires/VMProps.java` which isn't available in JDK 8. Have you tested this works?

I think something like this would be suitable here:

```
if (!Platform.is64Bit()) {
   System.out.println("Test requires 64 bits. Skipping...");
   return;
}
```

Reply:
Oh, you are right. I ran tests with 32-bit OSes enabled [1] and failure there disappeared, so I considered it working. I haven't realized test was also excluded on 64-bits. (condition was basically treated as always false) Nice find, I'll fix it. Thanks

[1] https://github.com/zzambers/jdk8u-dev/actions/runs/3471605048/jobs/5802167187

Reply:
But that reminds me, when I doing this backport, I  was actually wondering if it is supported on 8, so I grepped jdk8u-dev repo to see if there is some other tests doing this. There is one [2]. And it is indeed also skipped everywhere. So one more test to be fixed was discovered... :)

[2] https://github.com/openjdk/jdk8u-dev/blob/7ae002caba9ef17db5c105f76478424ce4cde04a/hotspot/test/runtime/Metaspace/MaxMetaspaceSizeTest.java#L29